luigi/interface.py

```diff
@@ -59,6 +77,8 @@ def setup_interface_logging(conf_file='', level_name='DEBUG'):
         stream_handler.setFormatter(formatter)
 
         logger.addHandler(stream_handler)
+    elif isinstance(conf_file, dict):
```

Comment:
I think we need to stay on this path. The reason I have to pass the config dictionary to luigi.setup_interface_logging() is to ensure that all of the subprocesses that luigi creates will have the correct logging configured. If I were to only call `logging.config.dictConfig(their_dict)` in my "driver" script then the logging config is not correctly set for the worker processes.

To re-clarify: I use a python "driver" script that calls luigi.run() to do my work.  If someone is using the luigi command directly or using python -m luigi, then manually calling `logging.config.dictConfig(their_dict)` might work, but I haven't tried or tested that.

I've updated my pull request to be squashed to a single commit on the current master branch and changed the variable name as you request.

Let me know if you have any questions.